### PR TITLE
[HMA-4267] Feedback from QA

### PR DESCRIPTION
**Pull request by tendai-moffatt**
Corrected button highlight colours

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Changed
 - Added optional body text to `InformationMessage`
 
+### Fixed
+- Fixed incorrect hex codes for button highlights in 5.4.1
```
Comment (paulb00th) [inline CHANGELOG.md:12]: The latest version of components is 5.5.0 so this will need to be 5.5.1

**Review by paulb00th — CHANGES_REQUESTED** [review]

File: CompanionApp/CompanionApp.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved [hunk 2]
```diff
@@ -6,8 +6,8 @@
         "repositoryURL": "https://github.com/hmrc/ios-components.git",
         "state": {
           "branch": null,
-          "revision": "0865e8a2414173cfe17c7ba98842f9b737ad35c8",
-          "version": "5.4.1"
+          "revision": "d9ea0d4d6a02fe81d05f6834f8ef99963f4ce6af",
+          "version": "5.5.0"
```
Comment (paulb00th) [inline CompanionApp/CompanionApp.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved:10]: This should be 5.5.1 too surely?

**Review by paulb00th — CHANGES_REQUESTED** [review]

File: Sources/UIComponents/Helpers/Colors/UIColor+Semantic.swift [hunk 3]
```diff
@@ -56,7 +56,7 @@ extension UIColor {
         public static var switchTintSelected = semanticColors.switchTintSelected
         public static var textInputBorder = semanticColors.textInputBorder
         public static var textInputLeftViewTint = semanticColors.textInputLeftViewTint
-        public static var transparentButtonHighlightedBackground = semanticColors.transparentButtonHighlightedBackground
+//        public static var transparentButtonHighlightedBackground = semanticColors.transparentButtonHighlightedBackground
```
Comment (paulb00th) [inline Sources/UIComponents/Helpers/Colors/UIColor+Semantic.swift:59]: I guess this can go now?

**Review by paulb00th — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (tendai-moffatt) [reply CHANGELOG.md:9]: This is to say that a bug found in version `5.4.1` has been fixed

**Review by paulb00th — APPROVED** [review]

**Pull request merged**